### PR TITLE
update sample qc summary notebook

### LINKED ISSUE #65: Improve results text: re sample QC
- Improve flow of text
- double check numbers because of exclusion of AG1000G-X samples.
- possible sankey diagram nh produced a while ago, for discourse?

### PR DESCRIPTION
**Pull request by hardingnj**
- deprecate ag3 module in favour of public
- split wild caught and crosses
resolves #65

### CODE REVIEW

File: content/02.results.md [hunk 1]
```diff
@@ -31,45 +31,40 @@ All samples successfully completed the pipeline and entered the sample quality c
...
-28 samples were excluded as they respectively dominated the first principal components, indicating high divergence from all other samples and therefore likely members of other Anopheline species.
-
+The third QC stage used principal component analysis (PCA) 
+to identify and exclude individual samples that were outliers based on available metadata.
+A review process identified samples that could not be explained parsimoniously, 
+and were therefore likely to be sample mix ups or instances of mislabelling.
+28 samples were excluded as they respectively dominated the first principal components, 
+indicating high divergence from all other samples and therefore likely members of other Anopheline species.
 A further 82 samples were excluded as potential sample mix ups.
+Following all sample QC steps, 2,874 wild-caught samples (70.2%) were retained for analysis.
```
Comment (alimanfoo) [inline content/02.results.md]:
```suggestion
Following all sample QC steps, 2,784 wild-caught samples (70.2%) were retained for analysis.
```

File: content/02.results.md [hunk 2]
```diff
@@ -31,45 +31,40 @@ All samples successfully completed the pipeline and entered the sample quality c
...
+was subject to a slightly different QC process.
+Firstly an analysis based on rates of Mendelian error identified true fathers of crosses 
+(where multiple males were introduced to cages), and validated provided pedigrees.
+Of the 729 samples sequenced we were able to validate 15 independent crosses to a high level of confidence, comprising 299 samples.
 4 of these crosses are novel relative to phase 2.
+These samples went through a modified sequence quality assurance process, 
+a single sample was removed for potential contamination (methods).
 
-These samples went through a modified sequence quality assurance process, where 1 sample was removed for insufficient coverage (methods).
+The final data release therefore comprises 3,081 samples, 297 from laboratory crosses, and 2,874 wild collected samples.
```
Comment (alimanfoo) [inline content/02.results.md]:
```suggestion
The final data release therefore comprises 3,081 samples, 297 from laboratory crosses, and 2,784 wild collected samples.
```

**Review by alimanfoo — COMMENTED** [review]
Couple of typos.

File: content/02.results.md (hunk 2 above)
Comment (hardingnj) [inline content/02.results.md]: 7-8 transposed

**Pull request merged**